services/net/notification/Models/ITnoRazorLightEngine.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TNO.Services.Notification.Models
+{
+    public interface ITnoRazorLightEngine
```

Comment:
Not a model, place code in root of project.

Reply:
Updated, thanks